### ISSUE TITLE
--blank and --blank-with-db should create an empty htdocs folder

### ISSUE THREAD

**Issue by bradp** [bradp/vv#141]

**Comment by neilgee** [comment]
Trying this I get a blank db, the site directory with the vvv-hosts file but **no** htdocs directory, vvv-init.sh or wp-cli.yml (eventhough the output says it did) - using vv version 1.8.0
```
neilg@[~/dev]:(develop) vv create --blank-with-db
New VVV Site Setup 
 Name of new site directory: mumba 
 Domain to use (leave blank for mumba.dev): 

About to perform the following: 
* Halt Vagrant (if running)
* Create directory mumba in /Users/neilg/dev/www
* Create files vvv-init.sh, wp-cli.yml, and vvv-hosts in directory mumba
* Create file mumba.conf in /Users/neilg/dev/config/nginx-config/sites
* Run `vagrant up --provision` to initialize site

Provisioning Vagrant will do the following: 

 
 Continue (y/n)? : y
[Success] New VVV setup starting for site 'mumba' 
==> default: Running triggers before halt...
==> default: Executing command "vagrant ssh -c vagrant_halt"...
==> default: VM must be running to open SSH connection. Run `vagrant up`
==> default: to start the virtual machine.
==> default: Command execution finished.
The command "vagrant ssh -c 'vagrant_halt'" returned a failed exit code. The
error output is shown below:

VM must be running to open SSH connection. Run `vagrant up`
to start the virtual machine.

Creating site directory, wp-cli.yml, and vvv-init.sh file...  
Done
Adding mumba.dev to new vvv-hosts file...  
Done
Creating nginx-config/sites/mumba.conf...  
Done
Running vagrant up --provision...  
```

**Comment by bradp** [comment]
Fixed in 44e491ae417d. Will be in next version.
